geode-core/src/main/resources/org/apache/geode/cache/doc-files/cache8_0.dtd

```diff
@@ -675,6 +675,7 @@ As of 6.5 roll-oplogs is deprecated. Use disk-store-name instead.
   alert-threshold              CDATA #IMPLIED
   dispatcher-threads           CDATA #IMPLIED
   order-policy                 CDATA #IMPLIED
+  group-transaction-events     (false | true) #IMPLIED
```

Comment:
I'm not sure the cache8_0.dtd should be changed. That implies that this property is supported in earlier versions.

Reply:
I was not sure of this. I recall seeing some failures in test cases and changed this and the cache7_0.dtd files that had been recently been updated.
I would appreciate if you can confirm me if these files or any of them should not be updated.

Reply:
I haven't gotten a definitive answer on this. I guess depending on the exceptions you've seen, I wouldn't update it or the 70 one.

Reply:
I finally figured out that it was not needed to change the dtd files. But I had to change 
CacheXmlGenerator so that the group-transaction-events attribute was not added to the gateway sender in case the cache.xml version was older than Geode 1.0. Without this change, the following test cases were failing:
```
org.apache.geode.cache.CacheXml80GatewayDUnitTest > testGatewaySenderWithSubstitutionFilter FAILED
    org.apache.geode.cache.CacheXmlException: While reading Cache XML file:/tmp/junit10230339244221650502/XML_8_0/CacheXml80GatewayDUnitTest_testGatewaySenderWithSubstitutionFilter.xml. Error while parsing XML, caused by org.xml.sax.SAXParseException; lineNumber: 4; columnNumber: 458; Attribute "group-transaction-events" must be declared for element type "gateway-sender".

org.apache.geode.cache.CacheXml70GatewayDUnitTest > testParallelGatewaySender FAILED
    org.apache.geode.cache.CacheXmlException: While reading Cache XML file:/tmp/junit5316048290838435700/XML_7_0/CacheXml70GatewayDUnitTest_testParallelGatewaySender.xml. Error while parsing XML, caused by org.xml.sax.SAXParseException; lineNumber: 4; columnNumber: 422; Attribute "group-transaction-events" must be declared for element type "gateway-sender".

org.apache.geode.cache.CacheXml70GatewayDUnitTest > testSerialGatewaySender FAILED
    org.apache.geode.cache.CacheXmlException: While reading Cache XML file:/tmp/junit13002812494872484215/XML_7_0/CacheXml70GatewayDUnitTest_testSerialGatewaySender.xml. Error while parsing XML, caused by org.xml.sax.SAXParseException; lineNumber: 4; columnNumber: 423; Attribute "group-transaction-events" must be declared for element type "gateway-sender".
```